### PR TITLE
update manifest

### LINKED ISSUE #174: Some packages in Manifest are outdated
Some packages in Manifest.toml under examples are outdated, such as EnsembleKalmanProcesses. It leads to an error when running the Lorenz example:

```
LoadError: MethodError: no method matching ParameterDistribution(::Vector{Dict{String, Any}})
Closest candidates are:
  ParameterDistribution(::Union{Array{PDType}, PDType}, ::Union{Array{CType}, Array, CType}, ::Union{Array{ST}, ST}) where {PDType<:ParameterDistributionType, CType<:EnsembleKalmanProcesses.ParameterDistributions.ConstraintType, ST<:AbstractString} at ~/.julia/packages/EnsembleKalmanProcesses/OjLuD/src/ParameterDistributions.jl:167
Stacktrace:
 [1] top-level scope
   @ /central/home/zhaoyi/CalibrateEmulateSample.jl/examples/Lorenz/Lorenz_example.jl:123
 [2] include(fname::String)
   @ Base.MainInclude ./client.jl:451
 [3] top-level scope
   @ REPL[2]:1
in expression starting at /central/home/zhaoyi/CalibrateEmulateSample.jl/examples/Lorenz/Lorenz_example.jl:108
```

### PR DESCRIPTION
**Pull request by szy21**
Closes #174

### CODE REVIEW

**Comment by odunbar** [comment]
seems like your manifest update breaks the linux build. Could you try just deleting the manifest instead?

**Comment by odunbar** [comment]
Strange - not seen this error before on the linux tests. It also doesn't come from your manifest changes as you only modify the example ones... rerunning

**Comment by odunbar** [comment]
see issue #176

File: examples/Emulator/GaussianProcess/Manifest.toml [hunk 1]
```diff
@@ -106,22 +106,22 @@ uuid = "83423d85-b0ee-5818-9007-b63ccbeb887a"
 version = "1.16.1+1"
 
 [[deps.CalibrateEmulateSample]]
-deps = ["AbstractMCMC", "AdvancedMH", "Conda", "Distributions", "DocStringExtensions", "EnsembleKalmanProcesses", "GaussianProcesses", "LinearAlgebra", "MCMCChains", "Printf", "PyCall", "Random", "ScikitLearn", "Statistics", "StatsBase"]
-path = "../../.."
+deps = ["AbstractMCMC", "AdvancedMH", "Conda", "Distributions", "DocStringExtensions", "EnsembleKalmanProcesses", "GaussianProcesses", "LinearAlgebra", "MCMCChains", "Pkg", "Printf", "PyCall", "Random", "ScikitLearn", "Statistics", "StatsBase"]
+git-tree-sha1 = "5a93353fdc751a2116529e907316cc51275b77b8"
```
Comment (charleskawczynski) [inline examples/Emulator/GaussianProcess/Manifest.toml:110]: We probably don’t want to add CES in this way, deving makes more sense

**Comment by odunbar** [comment]
@szy21 Hey! Could you resolve Charlie's comment, merge and try bors again - should be working now after PR 177

**Comment by szy21** [comment]
> @szy21 Hey! Could you resolve Charlie's comment, merge and try bors again - should be working now after PR 177

Yep, I just came back. Will do it today!

File: examples/Emulator/GaussianProcess/Manifest.toml (hunk 1 above)
Comment (szy21) [reply examples/Emulator/GaussianProcess/Manifest.toml:110]: Do you mean just reverting this change?

**Comment by tsj5** [comment]
Apologies if I'm on a different page here, but I think this PR isn't needed any more now that #177 was merged -- I wrote #177 by taking all of @szy21 's Manifest updates and pinning the sklearn version, so 177 should contain everything @szy21 has added.

The mistake @charleskawczynski caught above (not referring to a local, `dev`'ed version of CES) was fixed in https://github.com/CliMA/CalibrateEmulateSample.jl/commit/731abcd7527f843c62a9c74f5a69035e18a2801f (now squashed as part of #177).

**Comment by szy21** [comment]
Oh ok, I just came back and need to catch up. Thanks @tsj5! I will close it then.

**Pull request closed without merging**